incubation.md

```diff
@@ -0,0 +1,79 @@
+# Jupyter Incubation Process
+
+Incubation is the process by which new Jupyter Subprojects are created and evaluated for inclusion in the main Jupyter organization. In this context a Subproject is a GitHub repository with a development team that follows the Jupyter governance and contribution model. The goals of the incubation process are:
+
+* Contributors can quickly and easily get their code exposed to the Jupyter community while complying with corporate contribution restrictions.
+* Contributors can work with the community and gather feedback that will help them develop and refine a clear and concise integration proposal.
+* The broader user community can easily distinguish between stable core project feature functions and what is on the docket for potential future integration.
+* Enables the core development team to assess the readiness an applicability of proposed technologies against a backdrop of the broader project design efforts. 
+* Allows the core team to re-evaluate existing jupyter org repos to determine if any need to be moved to jupyter-incubator.
+
+## Proposal for incubation
+
+A subproject enter into incubation by submitting a proposal for incubation. This process is designed to be lightweight and quick:
+
+1. The proposers should fill out incubation application by sumitting a pull request to the
+   jupyter-incubator/proposals repository.
+2. The proposed subproject must have a mentor/sponsor/champion who is an active Steering 
+   Council member.
+3. The proposal for incubation will be discussed by the community and approved or rejected
+   by the consensus of the Steering Council.
+4. If approved the proposal PR will be merged, if rejected, it will be closed.
+
+## Incubation period
+
+Once a project has become an incubation subproject, the following steps will be taken:
+
+1. A GitHub repo will be created under the jupyter-incubator organization
+2. A GitHub team will created with read/write accees to the repo, including the
+   mentor/sponsor/champion.
+3. Standard Jupyter LICENSE file will be added to the repo.
+
+The goal of the incubation period is to demonstrate that the subproject is going to take off
+with an active developer and user community. There is no particular length of time required,
+but most projects will be in incubation for at least 6 months to 1 year.
```

Comment:
Another goal is to make sure the project develops into something that we think should be in the Jupyter core project.